### PR TITLE
Contentstream: Check for valid object or operand following EI in inline image parsing

### PR DESCRIPTION
**Pull request by gunnsth**
Addresses #469 by checking up to 3 objects/operands following EI to ensure that it is not part of the ID ... EI inline image data.

### CODE REVIEW

File: pdf/contentstream/inline-image.go [hunk 1]
```diff
@@ -437,14 +438,41 @@ func (csp *ContentStreamParser) ParseInlineImage() (*ContentStreamInlineImage, e
...
+							if err != nil && err != io.EOF {
+								return nil, err
+							}
+							dummyParser := NewContentStreamParser(string(peekbytes))
+
+							// Assume is done, check that the following 3 objects/operands are valid.
+							isDone := true
+							for i := 0; i < 3; i++ {
+								op, isOp, err := dummyParser.parseObject()
+								if err != nil && err != io.EOF {
```
Comment (adrg) [inline pdf/contentstream/inline-image.go]: Could avoid the extra check:
```
op, isOp, err := dummyParser.parseObject()
if err != nil {
	if err == io.EOF {
		break
	}
	continue
}
```

File: pdf/contentstream/inline-image.go [hunk 2]
```diff
@@ -437,14 +438,41 @@ func (csp *ContentStreamParser) ParseInlineImage() (*ContentStreamInlineImage, e
...
+									continue
+								}
+								if err == io.EOF {
+									break
+								}
+								if isOp && !isValidOperand(op.String()) {
+									isDone = false
```
Comment (adrg) [inline pdf/contentstream/inline-image.go]: Shouldn't there be a `break` here as well as it's already established that the peeked data does not contain valid operands?

**Comment by gunnsth** [comment]
@adrg Addressed the comments. Please take another look.

**Comment by adrg** [comment]
Looks great. Thanks for addressing the PR points.

**Pull request merged**